python/ray/services.py

```diff
@@ -957,12 +957,72 @@ def start_raylet(redis_address,
     return raylet_name
 
 
+def determine_plasma_store_config(object_store_memory=None,
+                                  plasma_directory=None,
+                                  huge_pages=False):
+    """Figure out how to configure the plasma object store.
+
```

Comment:
Can this docstring provide some more insight into what is going on?

Reply:
Done